WordPress/build.gradle

```diff
@@ -36,12 +36,19 @@ android {
         targetSdkVersion 23
 
         buildConfigField "String", "APP_PN_KEY", "\"org.wordpress.android.playstore\""
+        buildConfigField "boolean", "VISUAL_EDITOR_AVAILABLE", "false"
     }
 
     productFlavors {
-        vanilla {}
+        vanilla {} // used for release and beta
 
-        zbetagroup {
+        zalpha { // alpha version - enable experimental features
+            buildConfigField "boolean", "VISUAL_EDITOR_AVAILABLE", "true"
+            applicationId "org.wordpress.android"
```

Comment:
> Can't install the 2 versions side by side?

Yes this is intended since we release in the Play Store, we must use the same id, we also want to "force" our alpha testers to only use the latest version.